src/credentials_rotators/npm-token-rotation/README.md

```diff
@@ -0,0 +1,112 @@
+# NPM Token rotation
+
+This is a project that generates CDK stacks to rotate the NPM tokens. The stack is generated based on the configuration passed in `secret_config.json` file in the root of the project. The configuration should have the following shape
+
+#### Configuring the app
+
+The `secret_config.json` configuration file should be populated with the
+information about **accessing** secret values from AWS Secrets Manager. **It must not be
+used to store the secret values themselves.**
+
+## Sample `secret_config.json`
+
+```json
+{
+  "npmLoginUsernameSecret": {
+    "arn": "arn:aws:secretsmanager:us-east-2:12345678:secret:npm-username-secret-ui4F2W",
```

Comment:
We should redact any ARNs from the PR

Reply:
This isn't a real arn. If you look at the account number its some random string and other parts are also changed